src/FastRoute.php

```diff
@@ -133,29 +150,24 @@ public function withUriGenerator(string $uriGenerator): self
             $this->routeParser,
             $this->dataGenerator,
             $this->dispatcher,
-            $this->routesConfiguration,
+            $this->configureRoutes,
             $uriGenerator,
             $this->cacheDriver,
             $this->cacheKey,
         );
     }
 
     /** @return ProcessedData */
-    private function buildConfiguration(): array
+    public function processedConfiguration(): array
```

Comment:
This is meant to be encapsulated, please don't make it public

Reply:
> This is meant to be encapsulated

/me nods along

Is there a suggestion for how to inspect the processed routes, so that someone might (for example) dump a list of what paths map to what handlers?

Reply:
There are open issues about dumping that info and I was planning to introduce something that consolidates that info in a standard structure for debugging.